custom_components/hubitat/const.py

```diff
@@ -169,3 +170,9 @@ def __init__(
         H_CONF_UNLOCKED_WITH_CODE,
     ),
 }
+
+# Hubitat attributes that should be emitted as HA events
+_TRIGGER_ATTRS = tuple([v.attr for v in TRIGGER_CAPABILITIES.values()])
```

Comment:
Hmmm...alternatively, it may be fine to move them from hub.py to const.py, but in that case we don't really need `_TRIGGER_ATTRS`, just `_TRIGGER_ATTR_MAP` (and without the leading underscore). We would also want to addd an entry to `TRIGGER_CAPABILITES` for `RELEASABLE_BUTTON`, so button released events would work.